docs/deploy/s3.tf

```diff
@@ -41,7 +41,11 @@ resource "null_resource" "upload" {
   }
 
   provisioner "local-exec" {
-    command = "aws s3 sync ${var.build_dir}/docs-output/html s3://${aws_s3_bucket.docs.id}/${var.det_version} && aws cloudfront create-invalidation --distribution-id ${aws_cloudfront_distribution.distribution.id} --paths '/${var.det_version}/*'"
+    command = "aws s3 sync ${var.build_dir}/docs-output/html s3://${aws_s3_bucket.docs.id}/${var.det_version}"
```

Comment:
could also do a heredoc and just write it all like a script

Reply:
e.g. https://github.com/hashicorp/terraform/issues/3360#issuecomment-145393146

Reply:
https://www.terraform.io/docs/configuration/expressions.html#best-practices-for-dynamic-blocks

Looks like there is a special indented heredoc syntax of:
```
  command = <<-EOT
    cmd1 ;
    cmd2 ;
    cmd3
  EOT
```